app/scripts/controllers/eventDetails.js

```diff
@@ -449,6 +450,18 @@ angular
         validationErrorsHint = eventInformationPageHint;
       }
 
+      if (
+        $scope.conference.cruEvent &&
+        $scope.conference.ministry &&
+        $scope.getActivities().length !== 0 &&
+        !$scope.conference.ministryActivity
+      ) {
+        validationErrors.push(
+          'Please enter which Ministry Activitiy is applicable for this event.*',
```

Comment:
What's the `*`?

Reply:
I think it's to indicated that it's only applicable to certain ministries. All the other ```Ministry``` related validation errors have it so I just copied it from that https://github.com/CruGlobal/conf-registration-web/blob/7dcfbe90bd65bbd8dfee3c1bdf155a76c2ea2d29/app/scripts/controllers/eventDetails.js#L466